examples/dspy-evaluator/evaluator.py

```
@@ -0,0 +1,150 @@
+        content = req.content[:8000] if len(req.content) > 8000 else req.content
+
+        result = judge(content=content, file_path=req.artifactPath)
+
+        score = float(result.score) if isinstance(result.score, (int, float)) else 0.0
+        score = max(0.0, min(1.0, score))  # Clamp to [0, 1]
+
+        return EvalResponse(
+            score=score,
+            feedback=str(result.feedback) if result.feedback else None,
+        )
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)